src/components/todolist/styles.css

```diff
@@ -0,0 +1,46 @@
+.toast {
+  position: relative;
+  display: inline;
+  left: 990px;
```

Comment:
Instead of hard coding the width, we can use `left: 100%` as a starting point for the toast to be just out of the viewport and have display: 'none' CSS so that it is not visible by default

Reply:
Update the values accordingly in animation.